python/ray/actor.py

```diff
@@ -157,7 +157,7 @@ def _remote(self, args=None, kwargs=None, num_return_vals=None):
         if num_return_vals is None:
             num_return_vals = self._num_return_vals
 
-        def invocation(args, kwargs):
+        def invocation(args, kwargs, num_return_vals=num_return_vals):
```

Comment:
Was this change necessary?

Reply:
I needed to add this because the forward and backward invocations have a different number of return values. Is there a better way to do it?

Reply:
Yeah, that makes sense.